src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/IProjectContextInfo.cs

```diff
@@ -21,6 +21,7 @@ public interface IProjectContextInfo
         NuGetProjectKind ProjectKind { get; }
         ValueTask<bool> IsUpgradeableAsync(CancellationToken cancellationToken);
         ValueTask<IReadOnlyCollection<IPackageReferenceContextInfo>> GetInstalledPackagesAsync(CancellationToken cancellationToken);
+        ValueTask<IReadOnlyCollection<IPackageReferenceContextInfo>> GetTransitivePackagesAsync(CancellationToken cancellationToken);
```

Comment:
Maybe I need to stop being such an idealist, but it's odd to me that one PM UI action requires changes to two different `NuGet.VisualStudio.Internal.Contracts` classes, for a methods with the same name. How many network calls to the codespaces server is PM UI going to make for a single PM UI search? 😕

Reply:
Nothing wrong with being an idealist :) Btw, thank you for all of the comments. 
Is there a specific change you would like me to make here? I followed the pattern used by GetInstalledPackagesAsync.

Reply:
IProjectContextInfo no longer has any of these methods.